client/src/components/common/DropDown.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode } from 'react';
+import COLORS from 'src/constants/colors';
+import { fixedEntire, flexRow } from 'src/styles/common';
+import styled from 'styled-components';
+
+interface DropDownProps {
+  children: ReactNode;
+  close: () => void;
+}
+
+export default function DropDown({ children, close }: DropDownProps) {
+  const handleClick = (e: React.MouseEvent<HTMLDivElement>) => {
+    if (!(e.target as Element).className.includes('dropdown-container')) return;
+    close();
+  };
+
+  return (
+    <DropDownContainer className="dropdown-container" onClick={handleClick}>
+      <DropDownContent className="d">{children}</DropDownContent>
```

Comment:
이 클래스 이름은 사용되지 않는 것 같네요!
```suggestion
      <DropDownContent>{children}</DropDownContent>
```

Reply:
ㅎㅎ 그렇네요... 뭘까요 저건